cpp/include/cudf/column/column.hpp

```diff
@@ -213,6 +213,14 @@ class column {
    */
   bool has_nulls() const noexcept { return (null_count() > 0); }
 
+  /**
+   * @brief Indicates whether the column itself or its children columns contain null elements.
+   *
+   * @return true One or more elements are null
+   * @return false Zero elements are null
+   */
+  bool has_nested_nulls() const noexcept;
```

Comment:
Given that this is only used for implementing the `has_nested_nulls(table_view)` function, I'm not sure that this needs to be a member function.

Reply:
I think the functionality is very useful. Given that we are gradually increasing the usage of nested types, checking nested null for columns of data will be indispensable.

Reply:
We've had nested types for a long time now. Can you demonstrate other places it would be used? 

Adding new member functions to such a core data structure should not be done without significant consideration.

Reply:
I see. Thanks Jake, I'll reverse my change. @harrism we will not have such API then, as I concerned.

Reply:
Well you can still do the free function that operates on table view. My main concern was the name of your previous implementation. This is a better name.

Reply:
Alright, I have moved that free function into `table_view.hpp`  :relieved: